src/components/ProductCard/ProductCard.tsx

```diff
@@ -41,7 +45,8 @@ export default function ProductCard(props: ProductCardProps) {
   const { formatPrice } = state.formatters;
   const onClick = useOnProductCardClick(client, state.callbacks.onProductCardClick);
 
-  const cnstrcData = getProductCardCnstrcDataAttributes(productInfo);
+  const cnstrcData = getProductCardCnstrcDataAttributes(productInfo, item);
```

Comment:
```suggestion
  const cnstrcDataAttrs = getProductCardCnstrcDataAttributes(productInfo, item);
```
Maybe it's time for us to rename this const to reflect that it is for attributes?
This aligns with `addToCartBtnAttrs` as well